package.json

```diff
@@ -20,13 +23,14 @@
     "testAPI": "npm run build:dist && tape './tests/api/**/*.js'",
     "testAPI:browser": "npm run build:dist && karma start karma.conf.js",
     "test": "echo \"[INFO] Generic test cmd not used. See package.json for more specific test run cmds.\"",
-    "lint": "standard",
+    "tslint": "ethereumjs-config-tslint",
+    "tslint:fix": "ethereumjs-config-tslint-fix",
+    "lint": "ethereumjs-config-lint",
+    "lint:fix": "ethereumjs-config-lint-fix",
     "format": "ethereumjs-config-format",
-    "format-fix": "ethereumjs-config-format-fix",
-    "prepublishOnly": "npm run lint && npm run build:dist && npm run testBuildIntegrity",
-    "build:dist": "tsc",
-    "build:docs": "documentation build ./lib/index.js ./lib/runBlockchain.js ./lib/runBlock.js ./lib/runTx.js ./lib/runCode.js ./lib/runCall.js --format md --shallow > ./docs/index.md",
```

Comment:
The ``build:docs`` command has gotten lost, this is unintentional I suppose?

Reply:
Ah yeah, this was by mistake. I removed it to add the typedoc version (which I thought I had added :thinking:). Will re-add in the documentation PR.